### PR TITLE
Improvements for `var-spacing` in ansible-lint

### PR DESCRIPTION
**Pull request by ppuschmann**
The rule was formerly known as `jinja[spacing]`.

### CODE REVIEW

File: tasks/install_linux_repo.yml [hunk 1]
```diff
@@ -46,35 +46,33 @@
   become: true
 
 - name: Install repository
+  when: "ansible_os_family | lower in ['debian', 'redhat']"
+  become: true
```
Comment (nre-ableton) [inline tasks/install_linux_repo.yml:50]: This move is technically a fix for `key-order[task]`. I don't necessarily mind it being in this PR, but are you also working on fixing that linter warning?

File: tasks/main.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@
 
 - name: Include services management
   import_tasks: services.yml
-  when: consul_services is defined and consul_services|length>0
+  when: consul_services is defined and consul_services | length>0
```
Comment (nre-ableton) [inline tasks/main.yml]:
```suggestion
  when: consul_services is defined and consul_services | length > 0
```

I wish `ansible-lint` detected this, but it doesn't.

**Review by nre-ableton — CHANGES_REQUESTED** [review]
Great PR, thanks! :tada: However, before we merge it, please remove the `var-naming` suppression can be removed from `.ansible-lint`.

Thanks! If you don't have time to make the above changes, please let me know and I will fix (and force-push) to your branch.

File: tasks/install_linux_repo.yml (hunk 1 above)
Comment (ppuschmann) [reply tasks/install_linux_repo.yml:50]: As far as I can see, this is the last occurence of `key-order[task]`, am I right?

**Comment by nre-ableton** [comment]
@ppuschmann Thanks for responding to my comments. I'm really busy this week with a project that has a deadline before the Easter holidays, but I'll re-review your changes as soon as I can.

**Review by nre-ableton — APPROVED** [review]
Looks good! I'll merge this as soon as CI goes green. Thanks again for your contribution! 👍

File: tasks/install_linux_repo.yml (hunk 1 above)
Comment (nre-ableton) [reply tasks/install_linux_repo.yml:50]: I haven’t really looked, but I’ll make a separate PR to clean that up.

**Comment by nre-ableton** [comment]
CI problems have been fixed in https://github.com/ansible-community/ansible-consul/pull/547, but unfortunately that has led to some merge conflicts. Namely, the commit 84bb7cc6e19d0c8ee539262c47983825606152f0 ("Add pipefail to shell-tasks") must be dropped. See further explanation [here](https://github.com/ansible-community/ansible-consul/pull/547#issuecomment-1513570923) as to why we sadly can't use `set -o pipefail`, if you are curious.

So please rebase and drop the commit, and I'll go ahead and merge this PR.

Normally I would rebase this branch myself to save you the trouble, but I don't want to force-push your branch and delete your commits, so I would prefer if you could do it. 👍 If you aren't sure how to rewrite the history and drop the commit, just give me your blessing and I'll take care of it.

**Pull request closed without merging**